interface/themes/color_base.scss

```diff
@@ -479,14 +457,16 @@ div.section, div.borderbox {
 @import "colors/openemr5/codes";
 @import "colors/openemr5/bootstrap-nav-menu";
 @import "colors/openemr5/bootstrap";
-
 #globals_form_search {
-	margin-top: -3px;
-	padding: 2px 3px 1px 3px;
+    margin-top: -3px;
+    padding: 2px 3px 1px 3px;
 }
 th.currentvalues, th.historicalvalues {
-	background: $dark !important;
+    background: $dark !important;
 }
-
 @import "colors/openemr5/batch-payments";
 @import "colors/openemr5/help-files";
+@import "oe-common/help-files-common";
+@import "colors/openemr5/recall-flow-board";
+@import "oe-common/messages-common";
+@import "oe-common/acl-common";
```

Comment:
why aren't you using the import line discussed in README to bring in the 3 oe-common imports in 1 line?

Reply:
The stylesheets light and Manila are organized differently. The oe rules are added at the beginning and in the flat stylesheetss they are added at the end. Some rules are different because of consideration given to colors in the flat stylesheet

Reply:
I'm confused here as well, why aren't you just importing [all-common-import](https://github.com/zbig01/openemr/blob/zbig01-go-22/interface/themes/oe-common/all-common-import.scss) here? is there a reason why  `@import "colors/openemr5/recall-flow-board";` needs to be included between the three `oe-common` files?

Reply:
After thinking more about "The oe rules are added at the beginning", I have no idea why... None of the rules in [oe-boostrap](https://github.com/zbig01/openemr/blob/zbig01-go-22/interface/themes/oe-bootstrap.scss) are overridden within [style_light](https://github.com/zbig01/openemr/blob/zbig01-go-22/interface/themes/style_light.scss). My recommendation would be to:
1. Move oe-bootstrap into the oe-common folder. You can even rename it or further split it up into better-named sections, if you'd like.
2. then either:
- import oe-common into core (so it's inserted at the top of every scss file
- import oe-common into the bottom of each theme file (if that's necessary for your colors files, let's make that consistent across all the theme files)

Reply:
hi @zbig01  ,
Just checking what your response and plan is on the above posts?